projects/gloo/pkg/api/v1/enterprise/options/rbac/rbac.pb.go

```diff
@@ -312,7 +312,8 @@ type JWTPrincipal struct {
 	unknownFields protoimpl.UnknownFields
 
 	// Set of claims that make up this principal. Commonly, the 'iss' and 'sub' or 'email' claims are used.
-	// all claims must be present on the JWT.
+	// All claims must be present on the JWT. Nested claims, such as 'parent.child.foo', may also be used; however,
+	// the Policy must have a non-empty string value set for `nested_claim_delimiter`.
```

Comment:
same as above
I am not sure if these are generated here or from another file location, but adding a review here just in case
```suggestion
    // If you specify the path for a nested claim, such as 'parent.child.foo', you must also specify
    // a non-empty string value for the `nested_claim_delimiter` field in the Policy.
```

Reply:
It is indeed generated from `rbac.proto`, so I'll just re-run the codegen and push the changes, just to make sure there are no little differences